### PR TITLE
Security Monitor: remove link to notifications settings and add link to their wordpress account

### LINKED ISSUE #24937: Jetpack: Downtime Monitoring Notification Settings Redirect to JP Cloud
### Impacted plugin

Jetpack

### Steps to Reproduce

When a site uses the JP cloud instead of Calypso, the Downtime Monitoring Notifications Settings link is redirecting to JP cloud instead of Calypso.

1. Go to [site url]/wp-admin/admin.php?page=jetpack#/security. Make sure Downtime Monitoring is turned on and that the site in question is using JP Cloud instead of Calypso.
2. Click **Configure your notification settings**.

When I turned off JP cloud briefly, the redirect worked as expected.

### A clear and concise description of what you expected to happen.

It should go here: https://wordpress.com/settings/security/[site url]

### What actually happened

https://cloud.jetpack.com/settings/[site url]

### Other information

_No response_

### Reproducibility

Consistent

### Severity

_No response_

### Available workarounds?

_No response_

### Workaround details

_No response_

### Operating System

_No response_

### OS Version

_No response_

### Browser

Firefox

### Browser Version(s)

_No response_

### PR DESCRIPTION
**Pull request by miksansegundo**
Fixes #24937

Related to https://github.com/Automattic/dotcom-forge/issues/5655#issuecomment-1953522431 and p1708379298289459-slack-CDLH4C1UZ

## Proposed changes:

In the "Downtime Monitoring" card in Jetpack Settings > Security:
  * remove the remove link to the notifications settings
  * update the copy to include an external link to the user's WordPress account so they can change the email  
  * update the tracking event from `configure-monitor` to `configure-monitor-email`

|BEFORE|AFTER|
|-|-|
|Screenshot 2567-02-20 at 14 17 55|Screenshot 2567-02-20 at 14 20 49|

The link points to https://wordpress.com/me/account

Screenshot 2567-02-20 at 14 04 47

### Other information:

- [X] Have you written new tests for your changes, if applicable?
- [X] Have you checked the E2E test CI results, and verified that your changes do not break them?
- [X] Have you tested your changes on WordPress.com, if applicable (if so, you'll see a generated comment below with a script to run)?

## Does this pull request change what data or activity we track or use?
We update the tracking event from `configure-monitor` to `configure-monitor-email`.

Screenshot 2567-02-20 at 14 31 12

## Testing instructions:

* Go to `Settings > Security`
* Find the `Downtime Monitoring` option
* Verify the UI looks as expected and the link `WordPress account` opens your account in a new tab

### CODE REVIEW

**Review by fushar — APPROVED** [review]
Verified that I can see the new text, can click the link and it goes to the account settings, and that the correct event is fired.

**Comment by miksansegundo** [comment]
Thanks for testing @fushar!

Let's wait for a review from @Automattic/jetpack-vulcan as they work on the Jetpack monitor. cc: @lsl

**Review by jeherve — APPROVED** [review]
This tests well for me, it should be good to merge.

I created #35776 so we keep track of the idea of bringing those settings to wp-admin, and I updated this PR description so when merging, we will close #24937.

**Pull request merged**
